cardano-testnet/src/Testnet/Parsers.hs

```diff
@@ -29,19 +33,24 @@ data CardanoTestnetCommands
   | StartCardanoTestnet CardanoOptions
   | StartShelleyTestnet ShelleyOptions
   | GetVersion VersionOptions
+  | forall a. Help ParserPrefs (ParserInfo a) HelpOptions
```

Comment:
It's not obvious to me what we gain by making this an existential type. I think this is premature abstraction.

Reply:
Done!